README.md

```diff
@@ -94,6 +94,31 @@ __Maven__
 > **Note:** The devpack and compiler are only available for Neo N3. Thus, Java cannot be used to
 > compile smart contracts that are compatible with Neo Legacy.
 
+## Contribute
+
+Contributors are welcome!  
+
+Take a look at our open [issues](https://github.com/neow3j/neow3j/issues). We’re collecting all issues in the main neow3j repository, including issues related to documentation, tutorials, and coding examples. Some issues are labelled with a level of difficulty ([beginner](https://github.com/neow3j/neow3j/labels/beginner), [intermediate](https://github.com/neow3j/neow3j/labels/intermediate), [advanced](https://github.com/neow3j/neow3j/labels/advanced)). Create new issues if you want something not on the list.
+
+Join the AxLabs Discord server to chat with us and ask questions. Here are some points that should help you get started with the code base.
```

Comment:
Maybe we can add a link to join the Discord server here. @gsmachado can we create invite links that don't expire?

Reply:
oh yes I forgot. will add it.